src/popup/components/AccountDetailsNavigation.vue

```diff
@@ -76,18 +92,28 @@ export default defineComponent({
     Tab,
     BtnPill,
   },
+  props: {
+    isMultisig: Boolean,
+  },
   setup(props, { root }) {
+    const currentTabs = computed(() => {
+      if (props.isMultisig) {
+        return navigationConfig.filter((element) => !element.hideInMuiltisig);
+      }
+      return navigationConfig.filter((element) => !element.showOnlyInMultisig);
+    });
```

Comment:
Maybe instead of having one large array with boolean flags and then filtering into 2 separate ones create 2 separate ones by composing them from smaller blocks?

What I mean:
```
// First create collection of the nav items:
const navItemAssets: NavigationElement = {
  text: ...
  routeName: ...
}
const navItemTransactions: NavigationElement = {
  text: ...
  routeName: ...
}
const navItemNames: NavigationElement = {
  text: ...
  routeName: ...
  children: ...
}
...etc

// And then create two separate arrays:
const navigationConfigRegular: NavigationElement[] = [
  navItemAssets,
  navItemTransactions,
  navItemNames,
]
const navigationConfigMultisig: NavigationElement[] = [
  navItemAssets,
  navItemTransactions,
  navItemMultisigDetails,
]
```

Then in the setup:
```
const currentTabs = computed(() => (props.isMultisig) ? navigationConfigMultisig : navigationConfigRegular);
```

Reply:
good point